integration-tests/cypress/cypress.spec.js

```diff
@@ -1636,5 +1640,56 @@ moduleTypes.forEach(({
         })
       })
     })
+
+    // cy.origin is not available in old versions of Cypress
+    if (version === 'latest') {
```

Comment:
how is this going to behave for new releases? why not pin it to an specific version or a >=?
It would be nice to document why `latest` and not a version number.

Reply:
newer cypress releases will be tested. I could add a specific test for cypress `9.6.0`, where `cy.origin` was added, but I don't think it's worth it. We normally test our integrations in the oldest version we support and the newest one released